src/config/appDesignSystem/filter.ts

```diff
@@ -0,0 +1,44 @@
+import { consts } from '../consts';
+
+const { FILTER_TYPES } = consts;
+
+export const defaultFilterConfig = [
```

Comment:
is this needed with these null values?

Reply:
null or false. As a result, I think we need a default object with all filter options.

Reply:
but why? if there is nothing from the server there couldn't be a filter with such null values and `dataResourceType: 'dataResourceType',`. so from my logic there are filters, if the server is sending us configs, and there are not if the server does not send anything.

Reply:
I sent a new commit. Can you please check again?

https://github.com/smart-village-solutions/smart-village-app-app/pull/1188/commits/676bf433e9863a752a1b59a722cb4e6658c7ac4a